CODE_OF_CONDUCT.md

```diff
@@ -1,434 +0,0 @@
-ITK Community Participation Guidelines
```

Comment:
Instead, you could keep the file at the root, and consider `Documentation/docs/contributing/code_of_conduct.md` like this:

<pre>
```{include} ../../CODE_OF_CONDUCT.md
.md
```
</pre>

Reply:
This did not work as desired, at least for me.